resolwe_bio/processes/support_processors/gff_to_gtf.yml

```diff
@@ -41,10 +49,24 @@
     runtime: polyglot
     language: bash
     program: |
-      NAME=`basename {{annotation.annot.file}} .gff`
-      gff3_to_gtf.py {{ annotation.annot.file }} "${NAME}.gtf"
 
-      re-save-file annot "${NAME}.gtf"
+      {% set name = ((annotation.annot.file|basename) | replace(".gff3","") | replace(".gff","")) %}
+      gff3_to_gtf.py {{ annotation.annot.file }} {{name}}.gtf
```

Comment:
Not sure if this and the next few calls work if the file name includes spaces. Please refactor the test to have space in the file name.